src/commcare_cloud/commands/terraform/aws.py

```diff
@@ -583,8 +584,8 @@ def _has_valid_session_credentials_for_sso():
             continue
 
         if 'startUrl' in contents and 'expiresAt' in contents:
-            expiration = datetime.strptime(contents['expiresAt'], "%Y-%m-%dT%H:%M:%SUTC")
-            return datetime.utcnow() < expiration
```

Comment:
why remove this comparison? I think it checks whether the credentials are still valid so we do not need to auth again.

Reply:
See https://github.com/dimagi/commcare-cloud/pull/4455#issuecomment-759717915